src/components/subtitlesettings/subtitleappearancehelper.js

```diff
@@ -30,13 +30,13 @@ function getTextStyles(settings, preview) {
 
     switch (settings.dropShadow || '') {
         case 'raised':
-            list.push({ name: 'text-shadow', value: '-1px -1px white, 0px -1px white, -1px 0px white, 1px 1px black, 0px 1px black, 1px 0px black' });
+            list.push({ name: 'text-shadow', value: '-0.04em -0.04em  #FFF, 0px -0.04em #FFF, -0.04em 0px #FFF, 0.04em 0.04em #000, 0px 0.04em #000, 0.04em 0px #000' });
```

Comment:
Mostly the lower case is used. Also redundant space.
```suggestion
            list.push({ name: 'text-shadow', value: '-0.04em -0.04em #fff, 0px -0.04em #fff, -0.04em 0px #fff, 0.04em 0.04em #000, 0px 0.04em #000, 0.04em 0px #000' });
```
I'm not sure about this change (I don't like `Raised`/`Depressed`), but at least logically it looks right.